scss/_pagination.scss

```diff
@@ -1,31 +1,55 @@
 .pagination {
+  // scss-docs-start pagination-css-vars
+  --#{$variable-prefix}pagination-padding-x: #{$pagination-padding-x};
+  --#{$variable-prefix}pagination-padding-y: #{$pagination-padding-y};
+  --#{$variable-prefix}pagination-color: #{$pagination-color};
+  --#{$variable-prefix}pagination-bg: #{$pagination-bg};
+  --#{$variable-prefix}pagination-border-width: #{$pagination-border-width};
+  --#{$variable-prefix}pagination-border-color: #{$pagination-border-color};
+  --#{$variable-prefix}pagination-border-radius: #{$pagination-border-radius};
+  --#{$variable-prefix}pagination-hover-color: #{$pagination-hover-color};
+  --#{$variable-prefix}pagination-hover-bg: #{$pagination-hover-bg};
+  --#{$variable-prefix}pagination-hover-border-color: #{$pagination-hover-border-color};
+  --#{$variable-prefix}pagination-focus-color: #{$pagination-focus-color};
+  --#{$variable-prefix}pagination-focus-bg: #{$pagination-focus-bg};
+  --#{$variable-prefix}pagination-focus-box-shadow: #{$pagination-focus-box-shadow};
+  --#{$variable-prefix}pagination-active-color: #{$pagination-active-color};
+  --#{$variable-prefix}pagination-active-bg: #{$pagination-active-bg};
+  --#{$variable-prefix}pagination-active-border-color: #{$pagination-active-border-color};
+  --#{$variable-prefix}pagination-disabled-color: #{$pagination-disabled-color};
+  --#{$variable-prefix}pagination-disabled-bg: #{$pagination-disabled-bg};
+  --#{$variable-prefix}pagination-disabled-border-color: #{$pagination-disabled-border-color};
+  // scss-docs-end pagination-css-vars
+
   display: flex;
   @include list-unstyled();
 }
 
 .page-link {
   position: relative;
   display: block;
-  color: $pagination-color;
+  padding: var(--#{$variable-prefix}pagination-padding-y) var(--#{$variable-prefix}pagination-padding-x);
+  font-size: var(--#{$variable-prefix}pagination-font-size, inherit);
```

Comment:
Is the `inherit` fallback useful? Wouldn't it be the native fallback?

And don't we need RFS, here? We used the `font-size()` mixin, in `pagination-size()`.

Reply:
You're right, I've just removed the `inherit` fallback. I also added the `rfs()` mixin in the pagination sizing mixin.